docs/getting-started-guides/rkt/index.md

```diff
@@ -77,6 +77,89 @@ For more information on flannel configuration, please read [CNI/flannel README](
 Each VM on GCE has an additional 256 IP addresses routed to it, so it is possible to forego flannel in smaller clusters.
 This can most easily be done by using the builtin kubenet plugin, by setting the kubelet flag `--network-plugin=kubenet`.
 
+### Choose stage1 image
+
+rkt comes with the design that lets you to run the container images inside different isolation environments, which is known as [**rkt stage1**](https://coreos.com/rkt/docs/latest/devel/architecture.html#stage-1).
+Today, there are three coreos officially supported stage1s:
```

Comment:
I think we're going for 'supported by CoreOS (company)' not 'on CoreOS (distro)'.

My shot at it:

"The following three stage1s are supported by the rkt maintainers:"